templates/hoover-workers.nomad

```diff
@@ -61,6 +61,7 @@ job "hoover-workers" {
           # exec tail -f /dev/null
           if  [ -z "$SNOOP_TIKA_URL" ] \
                   || [ -z "$SNOOP_DB" ] \
+                  || [ -z "$SNOOP_THUMBNAIL_URL" ] \
```

Comment:
You don't need to add this to this `if` because it's not prone to disappear. This env is a constant (node IP + some path), as opposed to some others that originate from some `range service` loop that may sometimes have 0 iterations through it (e.g. when the database restarts). This means the env is always going to be set.